new/LIT7240PPAddam.xml

```diff
@@ -0,0 +1,76 @@
+                <title xml:lang="en" corresp="#t2">What was revealed (?) to Adam</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Magic prayer by virtue of the secret names revealed by God to <persName ref="PRS1389Adam">Adam</persName>, similar
+                    to <ref type="work" corresp="#LIT4576MagicPr "/> and <ref type="work" corresp="#LIT4600Salot "/> with the
+                    <term key="Asmat">ʾasmāt-word</term> <foreign xml:lang="gez">በኢያኤል</foreign>, which is also attested in 
+                    <ref type="work" corresp="LIT1758Lefafa"/> as well as in <ref type="mss" corresp="ESbmqm010#a2"/> and
+                    <ref type="mss" corresp="BMLor403#ms_i3"/>.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Magic"/>
+                    <term key="Prayers"/>
+                    <term key="Asmat"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2025-01-13">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="dc:relation" active="LIT7240PPAddam" passive="LIT4576MagicPr"><desc>Thematically similar to
+                        <ref type="work" corresp="LIT4576MagicPr"/>.</desc></relation>
+                    <relation name="dc:relation" active="LIT7240PPAddam" passive="LIT4600Salot"><desc>Thematically similar to
+                        <ref type="work" corresp="LIT4600Salot"/>.</desc></relation>
+                </listRelation>
+            </div>
+            <div type="edition">
```

Comment:
Can you mention what this edition is based on? I guess Strelcyn's description of a BL manuscript